gulpfile.js

```diff
@@ -19,7 +19,9 @@ const config = {
     typingsDir: path.join(__dirname, 'typings'),
     targetDir: path.join(__dirname, 'target'),
     fonts: path.join('fonts'),
-    moduleName: 'app'
+    moduleName: 'app',
+    assetsRepo: 'https://github.com/vblagoje/infinispan-management-console-assets.git',
```

Comment:
@ryanemerson this is just a test repo. We'll use a proper one when PR review is done